app/renderer/components/bookmarksToolbar.js

```diff
@@ -200,19 +204,43 @@ class BookmarkToolbarButton extends ImmutableComponent {
       onContextMenu={this.onContextMenu}>
       {
         this.isFolder && this.props.showFavicon
-        ? <span className='bookmarkFavicon bookmarkFolder fa fa-folder-o' style={iconStyle} />
+        ? <span className={css(
+            styles.bookmarkToolbarButton__bookmarkFavicon,
+            styles.bookmarkToolbarButton__bookmarkFolder
+          ) + ' ' + cx({
+            fa: true,
+            'fa-folder-o': true
+          })}
```

Comment:
I was thinking something like:
```
<span className={cx({
  fa: true,
  'fa-folder-o': true,
  [css(styles.bookmarkToolbarButton__bookmarkFavicon)]: true,
  [css(styles.bookmarkToolbarButton__bookmarkFolder)]: true
})}
```
This way you don't need `+ '  ' +` and it's more readable in my opinion